oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -788,6 +768,19 @@ protected void unbindNodeStateCache(DocumentNodeStateCache nodeStateCache) {
         }
     }
 
+    /**
+     * Closes the given array of features.
+     * <p>
+     * This method iterates over the provided array of features and closes each one
+     * that is not null.
+     * </p>
+     *
+     * @param features an array of {@link Feature} objects to be closed.
+     */
+    private void closeFeatures(final Feature[] features) {
```

Comment:
would using a vararg be an option? (would make the invocation call easier to write, less typing :)

Reply:
addressed in https://github.com/apache/jackrabbit-oak/pull/1845/commits/ed693368b9e6919d195c69b985375857814dea5d